public/locales/en-US/send.ftl

```diff
@@ -7,7 +7,7 @@ uploadPageLearnMore = Learn more
 uploadPageDropMessage = Drop your file here to start uploading
 uploadPageSizeMessage = For the most reliable operation, it’s best to keep your file under 1GB
 uploadPageBrowseButton = Select a file on your computer
```

Comment:
Do you actually need the string on the entity, or only on the title attribute? If it's the former, this should be

```
uploadPageBrowseButton
    .title = Select a file on your computer
```

Same question for similar strings.